src/unk_02074910.c

```diff
@@ -0,0 +1,17 @@
+#include "unk_02074910.h"
+
+BOOL SavArray_IsNatDexEnabled(SAVEDATA* savedata) {
+    return Pokedex_IsNatDexEnabled(Sav2_Pokedex_get(savedata));
+}
+
+BOOL Pokedex_IsNatDexEnabled(const POKEDEX* pokedex) {
+    return (Pokedex_GetNatDexFlag(pokedex) == 1);
+}
+
+u32 sub_02074930(BOOL natDexFlag, u32 species) {
+    if(natDexFlag == 0) {
+        return SpeciesToJohtoDexNo((u16) species);
```

Comment:
Style guide: No space between cast closing parenthesis and expression being cast
Style guide: No explicit cast required when passing an integer to a parameter of a narrower type. The only casts required in this project are between integer and pointer types, or between otherwise incompatible pointer types (`u8 *` to `struct FooStruct *`, or `const u8 *` to `u8 *`, etc.).

Reply:
Ahh thanks, this style guide helps since I (clearly) have a different style of writing code. 

Do you guys have a general file for a style guide or should I just keep a tab of all this advice somewhere for when I forget it in the future?